### DISCUSSION TITLE
Spring drinking_water ambiguous options

### DISCUSSION THREAD

**Discussion by pasbi** [streetcomplete/StreetComplete#4211] [Feedback]
StreetComplete recently asked me, if the water of a spring is drinkable.
Available options were approximately (originally in German and I don't remember them literally):

- not drinking water (with label)
- drinking water (with label)
- not drinking water (without label)
- drinking water (without label)

There was no label.
The more I think about the situation, the more I assume that in most cases, the absence of a label implies "not drinking water".
So maybe StreetComplete should display a hint to prefer "not drinking water (without label)" when in doubt.
If there is no label on the amenity, people in the field (including me) may be confused.

**Comment by 10992-osm** [comment]
You know, I was thinking about this after completing the quest for [node 9839682612](https://osm.org/node/9839682612).

I think there are cases where the answer is clearly "Not potable":

- Where the water is muddy or is surfacing in mud (as in the node above, surfacing in a patch of mud in a grassy field)
- Where the water otherwise appears unclean, or it is flowing out of something unclean.

Otherwise, it is more difficult (unless there is a sign).  If it's coming out of a spout, that suggests it's drinkable.

Perhaps checking `natural=spring` isn't worth it?

There's also the [wiki page](https://wiki.openstreetmap.org/wiki/Key:drinking_water).

See also:
#549

**Comment by pasbi** [comment]
Thanks for link #549, I wasn't able to find it.
Looks like there has been a discussion about this already.
I don't think it's worth starting another discussion.
